src/transforms/regex_parser.rs

```diff
@@ -12,28 +12,21 @@ use std::collections::HashMap;
 use std::str;
 use string_cache::DefaultAtom as Atom;
 
-#[derive(Deserialize, Serialize, Debug)]
+#[derive(Debug, Derivative, Deserialize, Serialize)]
+#[derivative(Default)]
 #[serde(default, deny_unknown_fields)]
 pub struct RegexParserConfig {
     pub regex: String,
     pub field: Option<Atom>,
+    #[derivative(Default(value = "false"))]
```

Comment:
```suggestion
    #[derivative(Default(value = "true"))]
```

Reply:
This follows the lead of other parser transforms like `json_parser` that default to `false`.

Reply:
Please update https://github.com/timberio/vector/blob/8c252a2c32fda4317284be46b70cf2e64e13d651/.meta/transforms/regex_parser.toml#L18 then! :)

Reply:
That would be the default for `drop_field`, which is a different setting.

Reply:
Yes, but in your branch:

![image](https://user-images.githubusercontent.com/130903/76367123-44d60480-62e9-11ea-8472-451f309369ba.png)

![image](https://user-images.githubusercontent.com/130903/76367138-57503e00-62e9-11ea-9372-3ddabde7e399.png)
In master:

![image](https://user-images.githubusercontent.com/130903/76367180-849cec00-62e9-11ea-8ec9-8ba30fa0c893.png)

https://github.com/timberio/vector/blob/9cd0ba6fca6b577c4c76956fc550496f5031175f/.meta/transforms/regex_parser.toml#L18

https://github.com/timberio/vector/blob/9cd0ba6fca6b577c4c76956fc550496f5031175f/src/transforms/regex_parser.rs#L30

Reply:
Ah I see it now, sorry. That was part of a lead up patch that I had forgotten about. Will fix.